### PR TITLE
Add namespace to CommitsTimeItem

### PR DESCRIPTION
**Pull request by yufeih**

### CODE REVIEW

File: src/docfx/build/metadata/CommitsTimeItem.cs [hunk 1]
```diff
@@ -4,11 +4,14 @@
 using System;
 using Newtonsoft.Json;
 
-internal class CommitsTimeItem
+namespace Microsoft.Docs.Build
```
Comment (OsmondJiang) [inline src/docfx/build/metadata/CommitsTimeItem.cs]: is there a run to valid?

Comment (yufeih) [reply src/docfx/build/metadata/CommitsTimeItem.cs]: I cannot find an existing stylecop rule that enforce that

**Comment by 928PJY** [comment]
what is this class used for?

**Comment by yufeih** [comment]
> what is this class used for?

A data model to resolve updated at based on git commit time

File: src/docfx/build/metadata/CommitsTimeItem.cs [hunk 2]
```diff
@@ -4,11 +4,14 @@
...
-    [JsonProperty("sha")]
-    public string Sha { get; set; }
+    internal class CommitsTimeItem
+    {
+        [JsonProperty("sha")]
+        public string Sha { get; set; }
 
-    [JsonProperty("built_at")]
-    public DateTime BuiltAt { get; set; }
+        [JsonProperty("built_at")]
```
Comment (live1206) [inline src/docfx/build/metadata/CommitsTimeItem.cs]: why not just use `[JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]`?

**Pull request merged**